docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,647 @@
+---
+type: docs
+title: "Tutorial: Use Helm to run your first app"
+linkTitle: "Helm application"
+weight: 200
+description: "Take a tour of Radius by updating an existing Helm chart to add Radius support."
+categories: "Tutorial"
+tags : ["kubernetes", "helm"]
+---
+
+This tutorial will teach you the following about Radius:
+
+* How to use update a Helm chart to include its resources in a Radius application
+* How to use features like Recipes and Connections within Kubernetes YAML or Helm.
+< image src="diagram.png" alt="Diagram of the application and its resources" width="500px" >
+
+## Prerequisites
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Helm](https://helm.sh/docs/intro/install/)
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+
+## Step 1. Clone and open the sample code
+
+{{< tabs "Codespace" "Local machine" >}}
```

Comment:
We can remove the codespaces option since there are issues with cluster creation

Reply:
commented out codespaces for now